Chinese/ZERO/ZERO.md.txt

```diff
@@ -0,0 +1 @@
+Click [here](http://www.google.com) to visit Google.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.